test/src/tests-event-logging.js

```diff
@@ -652,23 +773,28 @@ describe('event logging', function() {
         );
 
         done();
+        })
+        })
     });
 
     it('should log modify event', function(done) {
-        mockServer.respondWith(urls.modify, [
-            200,
-            {},
-            JSON.stringify({
-                change_results: [
+        waitForCondition(hasIdentifyReturned)
+        .then(() => {
+        fetchMockSuccess('https://identity.mparticle.com/v1/testMPID/modify', {
```

Comment:
Does this url not exist in constants? I think it possibly should but that can be a future task.

Reply:
it does.  i will update.